__tests__/app.test.js

```diff
@@ -12,4 +24,20 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
+
+  describe("GET /api/topics", () => {
+    test("200: Responds with an array of topic objects", () => {
+      return request(app)
+        .get("/api/topics") // requesting to endpoint
+        .expect(200) // expecting status 200
+        .then(({ body: { topics } }) => {
+          expect(topics).toBeInstanceOf(Array); // checking if topics is an array
```

Comment:
This isn't really needed since you know its an array because you're using a forEach on it and testing things within an array anyway. It's a good idea but its redundant.